src/System.Numerics.Vectors/tests/Vector2Tests.cs

```diff
@@ -327,20 +327,20 @@ public void Vector2ClampTest()
 
             // Case W1: specified value is in the range.
             a = new Vector2(0.5f, 0.3f);
-            expected = min;
```

Comment:
The fix, to get the NetFX tests passing is to take the modified bits of each test and split them out into their own test.

You should have one with the new behavior (`expected = max`) that is marked with `[SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]` (which is, skip on everything that isn't netfx) and you should have another with the original behavior (`expected = min`) that is marked with `[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]` (which is, skip only on netfx)